Source/SwiftLintFramework/Rules/RuleConfigurations/NameConfiguration.swift

```diff
@@ -17,6 +17,8 @@ public struct NameConfiguration: RuleConfiguration, Equatable {
     var minLength: SeverityLevelsConfiguration
     var maxLength: SeverityLevelsConfiguration
     var excluded: Set<String>
+    var allowedSymbols: Set<String>
```

Comment:
It wouldn't be difficult to add it to `TypeNameRule`and `GenericTypeNameRule`, but do you think it'll be a common case? I can't remember seeing any code base with types or generic types with symbols